src/Main.java

```diff
@@ -1,22 +1,35 @@
 import model.Elf;
 import model.Orc;
+import spawner.ElfSpawner;
+import spawner.OrcSpawner;
 
 public class Main {
     public static void main(String[] args) {
-        Orc orc = new Orc(100, 1, 3, 5);
-        Elf elf = new Elf(100, 2, 2, 4);
+        OrcSpawner orcSpawner = new OrcSpawner();
+        Orc orc = orcSpawner.spawnOrc();
+        ElfSpawner elfSpawner = new ElfSpawner();
+        Elf elf = elfSpawner.spawnElf();
 
+//        Orc orc = new Orc(100, 1, 3, 5);
```

Comment:
Please remove this comment, since orcs and elves are created by respective spawners